lib/api/controllers/adminController.js

```diff
@@ -116,6 +116,8 @@ function AdminController(kuzzle) {
       })
       .then(() => {
         kuzzle.pluginsManager.trigger('log:info', `Validation specification for ${modifiedRequestObject.index}#${modifiedRequestObject.collection} has been deleted.`);
+
+        return null;
```

Comment:
As of bluebird/promises specifications, if we dont care about the result of a promise, we need to use `return null` to let promises run in background

In this case, we don't care if the `log:info` event fail.
So yes, `return null` is better here